cumulusci.yml

```diff
@@ -281,6 +281,15 @@ tasks:
             path: unpackaged/config/trial
             namespace_tokenize: $project_config.project__package__namespace
 
+    deploy_Summer_21QA:
+        group: "EDA Metadata"
+        description: Deploys QA and Dev Only Config for Summer ‘21
+        class_path: cumulusci.tasks.salesforce.DeployBundles
+        options:
+            namespace_inject: $project_config.project__package__namespace
```

Comment:
The `namespace_inject` and `unmanaged` options are no longer required. CumulusCI can infer the needed configuration now!

Reply:
Thank you. This is fixed up in another branch. We have some interesting concurrent branch strategies at the moment that will exist only for this set of functionality.